### PR TITLE
Add nbstripout conda recipe

### PR DESCRIPTION
**Pull request by kynan**

### CODE REVIEW

**Comment by kynan** [comment]
The CircleCI failure is due to a network outage.

**Comment by kynan** [comment]
Is this now good to go?

File: recipes/nbstripout/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pytest-runner
+  run:
+    - python
+    - setuptools
```
Comment (jakirkham) [inline recipes/nbstripout/meta.yaml]: Why does this require `setuptools` at run time?

File: recipes/nbstripout/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+#   requires:
+#     - pytest
+#     - pytest-flake8
+#     - pytest-cram
+#   commands:
+#     - python setup.py test
+
+about:
+  home: https://github.com/kynan/nbstripout
+  license: MIT
```
Comment (jakirkham) [inline recipes/nbstripout/meta.yaml]: Would also suggest adding `license_family: MIT`.

File: recipes/nbstripout/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+  license_file: LICENSE.txt
+  summary: 'strip output from Jupyter and IPython notebooks'
+
+  # The remaining entries in this section are optional, but recommended
```
Comment (jakirkham) [inline recipes/nbstripout/meta.yaml]: Please drop this comment from the `example`.

File: recipes/nbstripout/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 8ca27ab35204b8352b376c071ed4543696d1dbb5088ab93dd282b134fdffd8fb
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
```
Comment (jakirkham) [inline recipes/nbstripout/meta.yaml]: Please add the following to `build`.

```
  entry_points:
    - nbstripout = nbstripout:main
```

File: recipes/nbstripout/meta.yaml (hunk 1 above)
Comment (jakirkham) [inline recipes/nbstripout/meta.yaml]: Assuming this is only require for the entrypoint. So please make the change above and drop `setuptools` from the `run` dependencies.

**Comment by kynan** [comment]
@jakirkham Thanks for the review, I hope I addressed your points.

**Comment by jakirkham** [comment]
Thanks @kynan .

Soon you should get an email that invite you to join conda-forge, @kynan . Once accepted you will be added to a team with the same name as this recipe. Those will give you permissions on the feedstock (repo) for this recipe.

Make sure when proposing any change that you go through the typical GitHub workflow of forking the feedstock and making changes in your fork that you PR back. Once merged CIs will build and deploy any changes you make.

Please let us know if you have any questions and welcome to conda-forge. :)

**Pull request merged**